preprocessing/nextclade/src/loculus_preprocessing/config.py

```diff
@@ -15,9 +15,24 @@
 CLI_TYPES = [str, int, float, bool]
 
 
+@dataclass
+class EmblInfoMetadataPropertyNames:
+    country_property: str = "geoLocCountry"
+    admin_level_properties: list[str] = dataclasses.field(
+        default_factory=lambda: ["geoLocAdmin1", "geoLocAdmin2", "geoLocCity", "geoLocSite"]
+    )
+    collection_date_property: str = "sampleCollectionDate"
+    authors_property: str = "authors"
+
+
 @dataclass
 class Config:
-    organism: str = "mpox"
+    organism: str = "cchf"
+    scientific_name: str = "Orthonairovirus haemorrhagiae"
+    molecule_type: str = "genomic RNA"
+    topology: str = "linear"
+    db_name: str = "Loculus"
+    # The backend host is the base API URL including the organism
```

Comment:
This makes sense of the comment above but if so it's misnamed as "host"? But maybe this is separate to this PR

Reply:
https://github.com/loculus-project/loculus/issues/4816